src/main/java/org/kohsuke/github/GHNotificationStream.java

```diff
@@ -180,7 +180,8 @@ GHThread fetch() {
             private long calcNextCheckTime() {
                 String v = req.getResponseHeader("X-Poll-Interval");
                 if (v==null)    v="60";
-                return System.currentTimeMillis()+Integer.parseInt(v)*1000;
+                long seconds = Integer.parseInt(v);
```

Comment:
In any case this code does not handle parsing exceptions, which may strike back at some point